mesa/space.py

```diff
@@ -989,22 +986,20 @@ def is_cell_empty(self, node_id: int) -> bool:
         return not self.G.nodes[node_id]["agent"]
 
     def get_cell_list_contents(self, cell_list: list[int]) -> list[GridContent]:
-        """Returns the contents of a list of cells ((x,y) tuples)
-        Note: this method returns a list of `Agent`'s; `None` contents are excluded.
+        """Returns the agents in a list of nodes; nodes with empty content are excluded.
```

Comment:
"the agents in a list of nodes" sounds ambiguous to me. It could mean 1. agents that are taken from the list of nodes, 2. agents, in the form of a list of nodes. Additionally, the argument name is `cell_list`, but the docstring says a list of nodes. I think saying cell contents instead of agents is fine, given that the 2nd sentence further reinforces that it is a list of `Agent`'s.

Reply:
The 2nd sentence could be removed once the return type annotation has been changed to `list[Agent]`.

Reply:
What about this wording? It seems to me better than the previous version and I think it addresses all your points. It seems to me that also, the second sentence can be removed the same even with the wrong return type like this

Reply:
I will lint as soon as I can do it